src/apps/answers/service.py

```diff
@@ -901,6 +901,12 @@ async def get_export_data(  # noqa: C901
                 respondent = subject_map[answer.target_subject_id]  # type: ignore
 
             answer.respondent_secret_id = respondent.secret_id
+            answer.source_secret_id = (
+                subject_map.get(answer.source_subject_id).secret_id if answer.source_subject_id else None  # type: ignore
+            )
+            answer.target_secret_id = (
+                subject_map.get(answer.target_subject_id).secret_id if answer.target_subject_id else None  # type: ignore
+            )
```

Comment:
In scenario where it is not an assessment from a reviewer respondent is the target_subject, so respondent_secret_id already target_secret_id, but it was not changed to not introduced a breaking change in api. we should review all fields name here not only add new ones (without introduce a breaking change), because we are duplicating data

Reply:
I understand. I added this field for the same reason: so that the change is backwards compatible. I think we can do such an assessment in the future, but for now we probably have to do it like this

Reply:
We already have a new ticket to handle his multiples names to same value in this endpoint response, let's go forward with this solution and refactor it later